### PR TITLE
folding@home yaml file with projects

### PR DESCRIPTION
**Pull request by mizimmer90**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: d9be4b1586a5a6edecd43d6f6fbd5c9224655136
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=22e94260-265f-4868-a799-27b11db5a09f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 0de4458a875c21f62d17452303090de0c7e6ced8
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=90c448fa-14dd-4751-9bcc-2fa21a34b99b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: fa8bb9b494eec1501516014e4f8fcbebfb3dbc9c
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=45e30f84-81e9-4175-885c-42ef5f0df8d2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: af541fa8bf357bbee1c544e31a8e510c9450ff75
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=7b96379d-f801-466b-bdfb-ef5ce2e38c03%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by chueatwork** [comment]
@mizimmer90 can you check the names of these buckets? Getting an error "Bucket `fah-public-data-covi19` doesn't exist"

File: datasets/foldingathome-covid19.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - COVID-19
+  - coronavirus
+  - simulations
+  - health
+  - life sciences
+  - protein
+License: CC0, https://creativecommons.org/share-your-work/public-domain/cc0/
+Resources:
+  - Description: Simulation of SARS-CoV-2 proteome and related. Emphasis on discovering cryptic pockets and functional dynamics.
+    ARN: arn:aws:s3:::fah-public-data-covi19
```
Comment (mcwitt) [inline datasets/foldingathome-covid19.yaml:18]: Is this meant to be:
```suggestion
    ARN: arn:aws:s3:::fah-public-data-covid19-cryptic-pockets
```

File: datasets/foldingathome-covid19.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: 
+    ARN: arn:aws:s3:::fah-public-data-covi19-absolute-free-energy
```
Comment (mcwitt) [inline datasets/foldingathome-covid19.yaml:22]:
```suggestion
    ARN: arn:aws:s3:::fah-public-data-covid19-absolute-free-energy
```

File: datasets/foldingathome-covid19.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: 
+    ARN: arn:aws:s3:::fah-public-data-covi19-relative-free-energy
```
Comment (mcwitt) [inline datasets/foldingathome-covid19.yaml:26]:
```suggestion
    ARN: arn:aws:s3:::fah-public-data-covid19-relative-free-energy
```

File: datasets/foldingathome-covid19.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: 
+    ARN: arn:aws:s3:::fah-public-data-covi19-antibodies
```
Comment (mcwitt) [inline datasets/foldingathome-covid19.yaml:30]:
```suggestion
    ARN: arn:aws:s3:::fah-public-data-covid19-antibodies
```

File: datasets/foldingathome-covid19.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: 
+    ARN: arn:aws:s3:::fah-public-data-covi19-moonshot-dynamics
```
Comment (mcwitt) [inline datasets/foldingathome-covid19.yaml:34]:
```suggestion
    ARN: arn:aws:s3:::fah-public-data-covid19-moonshot-dynamics
```

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 4a829732f1630075cb2756058400b935d79b1303
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=de2f68aa-ee8d-4fe3-a7c4-def93286b20c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: cd8f7ae5042007160f5629e16085eeda8c61e9cb
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f3add938-c63c-4d88-8708-f771977e1181%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 680225c33c89117782c0b796e86470945b39f689
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=be160307-e5be-4241-b948-100af3776664%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: a0ab90d0f72182cd00d1a4df03a8159c6007b14e
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=44dfa1f5-56b6-4dd2-ae8f-f8cb0e342d63%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: cd62b8da4767f30fe99759d04ef1dc78495f3818
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=66447b13-0c73-4936-b674-22a554e00cd9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e76c06bae823c4565b700d70adf437a771e0abf0
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=5a8f421a-df1c-4e0b-b2b2-e9adc0ee6129%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 1991ea59ae9429eb186664a888f40bd2361e4255
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=abc7f836-54ee-4ff6-ac70-9aaca4d65d93%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: fcd0a84b2d084e76ec28b47f665acd4371f2ee1f
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=18694313-82c4-42bf-9df0-79b029432ac6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by chueatwork — APPROVED** [review]

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e2469640dede4b9b5a8c8a550b36734b1f78b713
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=25efc07e-3401-4dbe-88e9-7c52a0cd8c8f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 8786fc6b2cd2fef4dac52241e55d0da255d24831
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=1e94f662-be3b-4ece-87a7-ab7c4fea05fb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 79a16d503dd37449f73d4afa50054a898edea1e3
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e31e9ac9-f1aa-4e3d-bef7-95adf6416718%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**